docker/Dockerfile_stack

```diff
@@ -238,10 +235,10 @@ RUN export TRILINOS_HASH=4c8adc92fcbad83b827f9f1b91a5060a4a9cdd65 && \
 ENV TRILINOS_DIR=/scratch/source/trilinos/release
 
 
-# download Kokkos-Tools
+# download Kokkos Profiling and Debugging Tools
 # sh does not support arrays so we need to use bash
-ENV KOKKOS_TOOLS_SHORT_HASH=7ee5bcb
-RUN ["/bin/bash","-c","export KOKKOS_TOOLS_HASH=7ee5bcb80a8329c8493da735bc7bad5a859dfb1f && \
+ENV KOKKOS_TOOLS_SHORT_HASH=2d75b69
```

Comment:
so we are pulling in this specific hash of Kokkos-Tools?

Reply:
Yes.  It is the current HEAD.